src/main/java/frc/robot/subsystems/IntakeNote/IntakeNote.java

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.IntakeNote;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+
+public class IntakeNote extends SubsystemBase {
+  /** Creates a new Intake. */
+  private final IntakeNoteIO intakeNoteIO;
+
+  private final IntakeNoteIOInputsAutoLogged intakeInputs;
+  // TODO: Change these speeds as needed once we test.
```

Comment:
is this resolved?

Reply:
No, we never got a prototype to test with cause it took too long and they just went to comp bot cad. So we will have to test speeds when we get the robot. But it shouldn't take too long.